svgdigitizer/electrochemistry/cv.py

```diff
@@ -96,10 +90,11 @@ def create_df_time_axis(self, df):
         df_['t'] = df_['cumdeltaU']/self.get_rate()
         return df_['t']
     
-    def plot_CV(self):
+    def plot_cv(self):
         self.df.plot(x='U_V', y='I_uA')
         plt.xlabel(f'{self.xlabel} / {self.xunit}')
         plt.ylabel(f'{self.ylabel} / {self.yunit}')
         
-    def create_csv(self):
-        self.df.to_csv(self.csvfile, index=False)
+    def create_csv(self, filename):
```

Comment:
I think we should actually just drop such functions and just document how to do this instead.